seerep_msgs/protos/query.proto

```diff
@@ -3,13 +3,11 @@ syntax = "proto3";
 package seerep.pb;
 
 import "time_interval.proto";
-import "boundingbox_stamped.proto";
 import "labels_with_category.proto";
 import "polygon2d.proto";
 
 message Query
 {
```

Comment:
Can you restructure this so that polygon is first and fullyEncapsulated second in the list? Please use the same order in fbs. Please also update the numbers